api/src/main/java/org/openmrs/module/patientqueueing/PatientQueueingUtil.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * <p>
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.patientqueueing;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PatientQueueingUtil {
+    public PatientQueueingUtil() {
+    }
+
+    /**
+     * Converts a delimited string to a list
+     * @param delimitedString  The string that contains items to be split
+     * @param delimiter the separator that will be used to split the sting into objects
+     * @return List a list of items after conversion
+     */
+    public static List delimitedStringToList(String delimitedString, String delimiter) {
```

Comment:
Isn't there a library that does this?

Reply:
@dkayiwa  not it doesn't but it would be a great add to the OpenmrsUtil

Reply:
Are you sure you did not get useful results by just googling with text like this? `java converting comma delimited string to list`

Reply:
Did you see the above comment?